### DISCUSSION TITLE
Theme Docsy - ERROR TOCSS: failed to transform "/scss/main.scss"

### DISCUSSION THREAD

**Discussion by nicfab** [google/docsy#2015] [Q&A] [answered]
Hi!
I updated Docsy running under `/themes/docsy` the command `git pull origin main`.
Now, if I run `hugo server` or `hugo server extended`, I receive the following error:

```
ERROR TOCSS: failed to transform "/scss/main.scss" (text/x-scss): "/Users/nfabiano/Documents/Hugo/privares/themes/docsy/assets/scss/_boxes.scss:121:10": no mixin named color-mode
Built in 2604 ms
Error: error building site: TOCSS: failed to transform "/scss/main.scss" (text/x-scss): "/Users/nfabiano/Documents/Hugo/privares/themes/docsy/assets/scss/_boxes.scss:121:10": no mixin named color-mode
```
Is there anyone who can help me?

**Comment by deining** [comment, marked as answer]
> Is there anyone who can help me?

Latest docsy versions have bootstrap version 5.3.x as dependency and it look like you are still on version 5.2.x.

Inside the directory `/themes/dosy`, run

``` 
npm install
```

and check if that cures your problem.

If not, run 

``` 
npm list
```

in the same directory and tell us the output.

**Side note**: You may consider migrating to hugo module, as described [here](https://www.docsy.dev/docs/updating/convert-site-to-module/).

**Reply by nicfab** [reply]
Thank you very much for your quick reply.
I ran `npm install`, and now it works.

**Comment by deining** [comment]
> Thank you very much for your quick reply. I ran `npm install`, and now it works.

Great!
Can you please mark your question as answered? Thanks!
